ports/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.7)
+project(antlr4-c3 VERSION 1.1.0)
```

Comment:
Considering the previous version as "1.0.0", "1.1.0" means the addition of CMake build.